data/schemas/com.github.akiraux.akira.gschema.xml.in

```diff
@@ -42,12 +42,6 @@
             <description>The saved width of the left panel. Must be greater than 220, or it will not take effect.</description>
         </key>
 
-        <key name="dark-theme" type="b">
-            <default>false</default>
-            <summary>Use dark theme</summary>
-            <description>Switch between Light and Dark theme.</description>
-        </key>
-
```

Comment:
You can't delete this.
Users need to be able to control the dark theme if they want a dark variation when they use a light OS theme.
We can't limit the app to only follow the OS.

Reply:
But when the app restarts it will follow the systems theme again

Reply:
Or if the systems theme changed the theme will changed too

Reply:
and I only removed the toggle not the setting

Reply:
You need to keep the toggle, removing it is wrong.

Reply:
but the theme will follow the system again when the app is restarted it will not take effect when the app is restarted
maybe we can add another toggle that asks use system theme variant

Reply:
Yes, let's do that!